api-reference/10 UI Widgets/dxDiagram/1 Configuration/nodes/autoLayout/type.md

```diff
@@ -5,10 +5,19 @@ type: String
 ---
 ---
 ##### shortDescription
-Specifies the diagram layout type.
+Specifies an auto-layout algorithm that is used to automatically arrange shapes.
 
 ---
- The **auto** value corresponds to **off** when both shape coordinates ([leftExpr](/api-reference/10%20UI%20Widgets/dxDiagram/1%20Configuration/nodes/leftExpr.md '/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#leftExpr') and [topExpr](/api-reference/10%20UI%20Widgets/dxDiagram/1%20Configuration/nodes/topExpr.md '/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#topExpr')) are specified; otherwise, the **auto** value corresponds to **layered**.
+- **auto**. Corresponds to **off** when both shape coordinates ([leftExpr](/api-reference/10%20UI%20Widgets/dxDiagram/1%20Configuration/nodes/leftExpr.md '/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#leftExpr') and [topExpr](/api-reference/10%20UI%20Widgets/dxDiagram/1%20Configuration/nodes/topExpr.md '/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#topExpr')) are specified; otherwise, the **auto** value corresponds to **layered**.
```

Comment:
corresponds?

Reply:
Is equivalent of **off** value when both shape coordinates ([leftExpr] and [topExpr]) are specified; otherwise, the **auto** value is equivalent of **layered**.

Reply:
The equivalent of the **off** value when both...